internal/protoc/compiler.go

```diff
@@ -203,14 +205,39 @@ func (c *compiler) runCmdMeta(cmdMeta *cmdMeta) ([]*text.Failure, error) {
 	c.logger.Debug("running protoc", zap.String("command", cmdMeta.String()))
 	buffer := bytes.NewBuffer(nil)
 	cmdMeta.execCmd.Stderr = buffer
-	// we only need stderr to parse errors
+	// We only need stderr to parse errors
 	// you have to explicitly set to ioutil.Discard, otherwise if there
-	// is a stdout, it will be printed to os.Stdout
+	// is a stdout, it will be printed to os.Stdout.
 	cmdMeta.execCmd.Stdout = ioutil.Discard
-	runErr := cmdMeta.execCmd.Run()
-	if runErr != nil {
-		// exit errors are ok, we can probably parse them into text.Failures
-		// if not an exec.ExitError, short circuit
+
+	// Prepare a signal buffer so that we can kill the protoc
+	// process when Prototool receives a SIGINT, SIGTERM,
+	// SIGQUIT or SIGHUP.
+	sig := make(chan os.Signal, 1)
+	done := make(chan error, 1)
+	signal.Notify(sig, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
```

Comment:
Sure, that seems reasonable. I included them here at first since they are generally handle-able and thought they should be passed along to `protoc`, but I'm fine with removing for now.

Reply:
This is semi-standard for all Unix binaries.